tensorboard/plugins/debugger_v2/tf_debugger_v2_plugin/views/graph_executions/graph_executions_component.css

```diff
@@ -58,6 +58,18 @@ limitations under the License.
   width: 100%;
 }
 
+.tensor-debug-info {
```

Comment:
I don't see any components using this in this PR but please always think about how `inline` element cannot have `block` element inside.